### PR TITLE
Chore/refactor mails

### PR DESCRIPTION
**Pull request by tsubik**
First step to refactor emails is to use ActionMailer properly with all its features like templating, previews, different adapters, testing etc.

Tested with Sendgrid API locally.

### CODE REVIEW

File: app/mailers/operator_mailer.rb [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+  # 1. Current transparency score
+  # 2. Change of score in the last quarter
+  # 3. List of documents expiring in the next quarter
+  # It's sent every quarter to all users of an operator
+  #
+  # TODO: Currently we don't have a way to know the language of the operator, so we send it in both French and English
+  # In the future this should be refactored:
+  # 1. To send it in the language of the user and use I18n
+  # 2. To send it using the html template
+  def quarterly_newsletter(operator)
```
Comment (santostiago) [inline app/mailers/operator_mailer.rb:41]: We'll have to update this one more time.

File: spec/factories/users.rb [hunk 2]
```diff
@@ -38,8 +38,12 @@
     name { 'Test user' }
     is_active { true }
 
-    after(:build) do |random_user|
-      random_user.user_permission = UserPermission.new(user_role: 0)
+    transient do
```
Comment (santostiago) [inline spec/factories/users.rb:41]: ❤️

**Review by santostiago — COMMENTED** [review]
@tsubik the mailer preview specs have no assertions. What is being tested?
All the rest looks good,! 👍

File: app/mailers/operator_mailer.rb (hunk 1 above)
Comment (tsubik) [reply app/mailers/operator_mailer.rb:41]: Ok. I just saw the newest PR.

**Comment by tsubik** [comment]
@santostiago those are not specs, but previews. Test folder is the default folder for mailer previews code. https://guides.rubyonrails.org/action_mailer_basics.html#create-the-mailer

File: app/mailers/operator_mailer.rb (hunk 1 above)
Comment (santostiago) [reply app/mailers/operator_mailer.rb:41]: Did you rebase and update it?

**Review by santostiago — APPROVED** [review]

File: app/mailers/operator_mailer.rb (hunk 1 above)
Comment (tsubik) [reply app/mailers/operator_mailer.rb:41]: Looks like this is the latest version, no? I copied it over from develop.

**Pull request merged**